.env.sample

```
@@ -1,6 +1,10 @@
 CLIENT_PORT=3000
 SERVER_PORT=3001
```

Чтоб работала авторизация через OAuth нужно чтоб локально сервер запускался на 3000 порту.
```suggestion
CLIENT_PORT=2999
SERVER_PORT=3000
```